src/emailpro/emailpro.controller.js

```diff
@@ -264,6 +264,7 @@ angular.module("Module.emailpro.controllers").controller("EmailProCtrl", [
                     case "todo":
                     case "DONE":
                     case "done":
+                    case "success":
```

Comment:
Make sure adding lowercase in swicth ?
(update cases after)

Reply:
Why not use `toLowerCase()` (or `toUpperCase()`) on `failure.type` instead of covering both cases ?

Reply:
Yes that's what I've done !